discovery-frontend/src/app/common/directive/explore-data-scroll.directive.ts

```diff
@@ -0,0 +1,15 @@
+import {Directive,     HostListener} from "@angular/core";
+
+@Directive({
+  selector: '[exploreDataScroll]'
+})
+export class ExploreDataScrollDirective {
+  // 이 함수 이름은 맘대로 해도된다.
```

Comment:
I'm sorry to bother you, but we decided to refuse the content in Korean.

Please change some comments in this PR.